lib/modulesync.rb

```diff
@@ -21,12 +24,16 @@ def self.config_defaults
     }
   end
 
+  def self.options
+    @options
+  end
+
   def self.local_file(config_path, file)
     File.join(config_path, MODULE_FILES_DIR, file)
   end
 
-  def self.module_file(project_root, namespace, puppet_module, *parts)
-    File.join(project_root, namespace, puppet_module, *parts)
+  def self.module_file(puppet_module, *parts)
+    File.join(puppet_module.working_directory, *parts)
   end
```

Comment:
Should this method be one in the `PuppetModule` as well? What triggered me was `module_file(puppet_module, MODULE_CONF_FILE)` which can probably be `puppet_module.path(MODULE_CONF_FILE)` or similar.

Reply:
You are right, its already planned in next PR. This one is a part of #206